packages/daemon/src/mail.js

```diff
@@ -97,37 +113,31 @@ export const makeMailboxMaker = ({
       return harden(names);
     };
 
-    /**
-     * @param {unknown} presence
-     */
+    /** @type {import('./types.js').Mail['reverseLookup']} */
     const reverseLookup = async presence => {
-      const formulaIdentifier = formulaIdentifierForRef.get(await presence);
+      const formulaIdentifier = getFormulaIdentifierForRef(await presence);
       if (formulaIdentifier === undefined) {
         return harden([]);
       }
       return reverseLookupFormulaIdentifier(formulaIdentifier);
     };
 
-    /**
-     * Takes a sequence of pet names and returns a formula identifier and value
-     * for the corresponding lookup formula.
-     *
-     * @param {string[]} petNamePath - A sequence of pet names.
-     * @returns {Promise<{ formulaIdentifier: string, value: unknown }>} The formula
-     * identifier and value of the lookup formula.
-     */
+    /** @type {import('./types.js').Mail['provideLookupFormula']} */
     const provideLookupFormula = async petNamePath => {
       // The lookup formula identifier consists of the hash of the associated
       // naming hub's formula identifier and the pet name path.
-      // A "naming hub" is an objected with a variadic lookup method. At present,
-      // the only such objects are guests and hosts.
+      // A "naming hub" is an objected with a variadic lookup method. It includes
+      // objects such as guests and hosts.
       const hubFormulaIdentifier = identifyLocal('SELF');
```

Comment:
Aside: I think this identifier is a lexical variable and doesn’t require `identifyLocal`.

Reply:
https://github.com/endojs/endo/pull/2085